src/themes/teams-high-contrast/components/Avatar/avatarVariables.ts

```diff
@@ -2,5 +2,7 @@ import { AvatarVariables } from '../../../teams/components/Avatar/avatarVariable
 import { Partial } from 'types/utils'
 
 export default (siteVariables: any): Partial<AvatarVariables> => ({
+  avatarBorderColor: 'white',
```

Comment:
would suggest to use color from `siteVariables` here (i.e. `siteVariables.white`), for the sake of consistency - or let me know if there any reason to not use it

Reply:
Good catch. Thanks!